### DISCUSSION TITLE
Images in Bootstrap5 accordions

### DISCUSSION THREAD

**Discussion by skopelosbill** [community/community#45151] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi people, I am a 68 year old newbie and have just uploaded my first website to github.  I have made it for a friend.

I have two accordions, one with text and one with text and images but the images are all broken links.  Is it OK to put images in an accordion?  Is there a special way of doing it?

I would appreciate any help with this and any comments at all on the site and on the code I have written, for much of which I am indebted to Iamshaun.github.io  (not sure yet how to link to another person on here).

my site is skopelosbill.github.io

**Comment by jge162** [comment]
Yes, it is perfectly fine to include images in an accordion. However, it sounds like there may be an issue with the way you have linked to the images in your accordion.

@skopelosbill One possible issue could be that the file path to the images is incorrect. Make sure that the file path to the images is correct and that the images are located in the correct directory.

Another possibility is that the images may not be included in the repository when you pushed it to GitHub.

it's great that you have started your journey in web development. If you have any question, don't hesitate to reach out here in forum.

**Reply by skopelosbill** [reply]
thanks @jge162.  I am going to have a good look at that now, one of the images is called isthmus so there is the possibility of a typo in that one somewhere but the other two should be obvious mistakes, but I haven't spotted them yet - I shall see, thanks for your encouragement.

**Comment by skopelosbill** [comment, marked as answer]
Aha!  I have now found the problem.  Most of the photographs in the site were linked like this

<img src="../Assets/isthmus.jpg"

but four of them should have been linked like this

<img src="../Assets/isthmus.JPG"

why I named them xyz.JPG I will never know, but I did so now everything I have so far put into my site is working.  What I need to work out now is how to change the names of directories and files within github if that is possible.

So thanks again to @jge162 for you direction.

**Reply by jge162** [reply]
sweet, good job
